README.md

```diff
@@ -105,4 +108,64 @@ module Mutations
     end
   end
 end
-```
\ No newline at end of file
+```
+
+### gql:search_object MODEL_NAME
+
+Generate a search object from a model using [SearchObjectGraphQL](https://github.com/RStankov/SearchObjectGraphQL)
```

Comment:
Probably worth adding a note here about adding the "search_object_graphql" gem to your Gemfile.

Reply:
The gem now gets added when the ModelSearchBaseGenerator is run.